### PR TITLE
fix(DB/gameobject_loot): Gordok Tribute

### PR DESCRIPTION
**Pull request by Knindzagxg**
Updated Gordok Tribute which is extra loot after killing King in Dire Maul.

Was missing every rare item.

Drop chance updated as well.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1598878453024493200.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (16577, 18533, 0, 7.1, 0, 1, 1, 1, 1, NULL);
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (16577, 18529, 0, 7.6, 0, 1, 1, 1, 1, NULL);
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (16577, 18528, 0, 8.9, 0, 1, 1, 1, 1, NULL);
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (16577, 18537, 0, 11.1, 0, 1, 1, 1, 1, NULL);
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (16577, 18499, 0, 10.8, 0, 1, 1, 1, 1, NULL);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1598878453024493200.sql:22]: DELETE & INSERT :)

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1598878453024493200.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1598878453024493200.sql:22]: Shin already poked me on Disc as well xP

Hes gonna sort this one due diff % on every damn item xP

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1598878453024493200.sql:22]: also 2020-08-31 14:31:02 Table 'gameobject_loot_template' entry 16577 group 1 has total chance > 100% (262.899994)

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1598878453024493200.sql:22]: Every group can have a max of 100%

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1598878453024493200.sql:22]: Yeah, was reading about it and got confused lol xD

Then just made for each item xP
Feel free to update SQL, i need to see example xD

**Comment by Remberance** [comment]
2020-09-01 12:25:50 Table 'gameobject_loot_template' entry 16577 group 1 has total chance > 100% (262.899994)

File: data/sql/updates/pending_db_world/rev_1598878453024493200.sql (hunk 1 above)
Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1598878453024493200.sql:22]:
```sql
DELETE FROM `gameobject_loot_template` WHERE (`Entry` = 16577);
INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
(16577, 1, 12007, 100, 0, 2, 0, 1, 1, NULL),
(16577, 2, 12007, 100, 0, 4, 0, 1, 1, NULL),
(16577, 3, 12007, 100, 0, 8, 0, 1, 1, NULL),
(16577, 4, 12007, 100, 0, 16, 0, 1, 1, NULL),
(16577, 5, 12007, 100, 0, 32, 0, 1, 1, NULL),
(16577, 8766, 0, 100, 0, 1, 0, 15, 20, NULL),
(16577, 8952, 0, 100, 0, 1, 0, 15, 20, NULL),
(16577, 13444, 0, 75, 0, 1, 1, 2, 5, NULL),
(16577, 13446, 0, 80, 0, 1, 1, 2, 5, NULL),
(16577, 18495, 0, 9.1, 0, 1, 1, 1, 1, NULL),
(16577, 18499, 0, 10.8, 0, 1, 1, 1, 1, NULL),
(16577, 18500, 0, 7.9, 0, 1, 1, 1, 1, NULL),
(16577, 18528, 0, 8.9, 0, 1, 1, 1, 1, NULL),
(16577, 18529, 0, 7.6, 0, 1, 1, 1, 1, NULL),
(16577, 18530, 0, 9.9, 0, 1, 1, 1, 1, NULL),
(16577, 18531, 0, 8.2, 0, 1, 1, 1, 1, NULL),
(16577, 18532, 0, 10.5, 0, 1, 1, 1, 1, NULL),
(16577, 18533, 0, 7.1, 0, 1, 1, 1, 1, NULL),
(16577, 18534, 0, 9.3, 0, 1, 1, 1, 1, NULL),
(16577, 18537, 0, 11.1, 0, 1, 1, 1, 1, NULL),
(16577, 18655, 0, 7.5, 0, 1, 1, 1, 1, NULL);
```

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1598878453024493200.sql]: Updated to Shin's "format".

Should be ok now, i think.

**Comment by Knindzagxg** [comment]
@Si1ker or @Kitzunu can you check SQL now.
Just updated with things Shin generated with K3.

**Comment by FrancescoBorzi** [comment]
@Knindzagxg 

```
2020-09-28 15:41:29 Table 'gameobject_loot_template' entry 16577 group 1 has total chance > 100% (262.900024)
```

**Comment by Knindzagxg** [comment]
Im not sure how to change it tbh, never tried with gameobject loot.
Any example with link?

**Comment by FrancescoBorzi** [comment]
@Knindzagxg it's not about gameobject, it's the same for all loots.

Basically when you have a group loot, then the sum of all items belonging to that group cannot be more than 100.

In Keira3 if you open the loot, you'll see the colored column with the group number. Different groups have different colours so it's more readable there. Make sure every group does not have more than 100 (as the sum of their items).

**Comment by Knindzagxg** [comment]
Think i got ya.
Ill need to check values again and to work and get correct ones for whole group.

**Comment by Knindzagxg** [comment]
Think it got now @FrancescoBorzi

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Racknar0** [comment]
Tested!

image

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**